include/rocksdb/write_buffer_manager.h

```diff
@@ -46,16 +48,19 @@ class WriteBufferManager {
   size_t dummy_entries_in_cache_usage() const {
     return dummy_size_.load(std::memory_order_relaxed);
   }
-  size_t buffer_size() const { return buffer_size_; }
+  size_t buffer_size() const {
+    return buffer_size_.load(std::memory_order_relaxed);
+  }
 
   // Should only be called from write thread
   bool ShouldFlush() const {
     if (enabled()) {
       if (mutable_memtable_memory_usage() > mutable_limit_) {
```

Comment:
mutable_limit_ is based on the initial value of buffer_size.  Should it also be changed if the buffer size is changed?

Reply:
unclear how I missed this.  thank you. done.